index.js

```diff
@@ -0,0 +1,47 @@
+let fs = require('fs');
+let solc = require('solc');
+let path = require('path');
+
+let contractCache;
+
+let WeTrustContract = {
+  /**
+   * Call this function upon server start. It reads and compiles the contract, and saves it in a cache variable.
+   * Throws if there was an error.
+   */
+  init: function(done) {
```

Comment:
The init() is still in this index.js.  We call it before export so that client does not have to make extra call.  Basically, something like this:

let init = function(){...};
init();
let WeTrustRosca = {...};
exports = WeTrustRosca